clients/drcachesim/common/directory_iterator.cpp

```diff
@@ -128,8 +129,24 @@ directory_iterator_t::is_directory(const std::string &path)
 }
 
 bool
-directory_iterator_t::create_directory(const std::string &path)
+directory_iterator_t::create_directory(const std::string &path_in)
 {
-    drfront_status_t res = drfront_create_dir(path.c_str());
-    return res == DRFRONT_SUCCESS;
+    std::string path = path_in;
```

Comment:
nit: Do we need this local variable? Can't we use `path_in` directly?

Reply:
path_in is const so the Windows replacement fails without a writable local copy.